src/main/msp/msp.c

```diff
@@ -1500,13 +1500,13 @@ static bool mspProcessOutCommand(int16_t cmdMSP, sbuf_t *dst)
         sbufWriteU8(dst, 0);
 #endif
         sbufWriteU8(dst, rxConfig()->fpvCamAngleDegrees);
-        sbufWriteU8(dst, rxConfig()->rcInterpolationChannels);
+        sbufWriteU8(dst, 0); //was sbufWriteU8(dst, rxConfig()->rcSmoothingChannels);
```

Comment:
Needs space after `//`. We don't need the `sbufWriteU8(dst, ` part in the comment.
It's very helpful for those maintaining the configurator and lua scripts if the comment says in which MSP API version the parameter was removed. For 4.3 it's 1.44